bootstrap_test.py

```diff
@@ -280,6 +280,28 @@ def test_consistent_range_movement_true_with_rf1_should_fail(self):
     def test_consistent_range_movement_false_with_rf1_should_succeed(self):
         self._bootstrap_test_with_replica_down(False, rf=1)
 
+    def test_RF_GT_Nodes_should_succeed(self):
+        cluster = self.cluster
+        cluster.set_environment_variable('CASSANDRA_TOKEN_PREGENERATION_DISABLED', 'True')
+        cluster.populate([2, 2])
+        cluster.start()
+
+        node1 = cluster.nodelist()[0]
+        session = self.patient_cql_connection(node1)
+        session.execute("CREATE KEYSPACE k WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'dc1' : '3'}")
+
+        node5 = Node('node5',
+                     cluster,
+                     True,
+                     ('127.0.0.5', 9160),
+                     ('127.0.0.5', 7000),
+                     '7500',
+                     '0',
+                     None,
+                     binary_interface=('127.0.0.5', 9042))
+        cluster.add(node5, is_seed=False, data_center="dc1")
```

Comment:
Probably too much, but
should we add a failed read before / successful read after addition?

Reply:
It won't hurt...